.travis.yml

```diff
@@ -24,10 +24,22 @@ before_install:
   - rm -rf "$HOMEBREW_TAP_DIR"
   - ln -s "$PWD" "$HOMEBREW_TAP_DIR"
   - export HOMEBREW_DEVELOPER="1"
+  - export HOMEBREW_BINTRAY_USER="davidchall"
+  - export TAP_BOTTLE_DOMAIN="https://dl.bintray.com/davidchall"
   - ulimit -n 1024
 
 script:
-  - brew test-bot
+  - brew tap davidchall/test-bot
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+      brew test-bot --ci-testing;
+    else
+      brew test-bot --ci-pr;
+    fi
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" ]; then
```

Comment:
Could maybe just do/detect this on any non-fork-PR branch?

Reply:
When to upload the bottle is a matter of workflow. I think we should discuss this when I open a PR on the test-bot repo. For now, I just want a proof of concept.